pkl-core/src/main/java/org/pkl/core/EvaluatorImpl.java

```diff
@@ -280,6 +280,7 @@ String evaluateOutputText(VmTyped fileOutput) {
     return doEvaluate(() -> VmUtils.readTextProperty(fileOutput));
   }
 
+  @SuppressWarnings({"removal", "RedundantSuppression"})
```

Comment:
What is this suppressing? I don't see these warnings locally after removing this annotation.

Reply:
```
} catch (ThreadDeath e) {
```
... is [deprecated for removal](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/ThreadDeath.html) since JDK 20. Compiling under earlier versions would therefore need a `RedundantSuppression` as well

Reply:
Ah, I see!

It's been a while since I've looked at this code, but, I think we can actually change that to `catch (Throwable e)`.